Utilities/DataFlow/CMakeLists.txt

```diff
@@ -94,5 +97,8 @@ O2_GENERATE_TESTS(
 
 O2_GENERATE_MAN(NAME TimeframeReaderDevice)
 O2_GENERATE_MAN(NAME TimeframeWriterDevice)
+O2_GENERATE_MAN(NAME SubframeBuilderDevice)
 
 target_compile_options(test_TimeframeParser PUBLIC -O0 -g)
```

Comment:
We should have both optimised and non optimised, as I've already said. I think it's up to WP3 to find and implement a solution for that. Tests are not only there to ensure bugs are not reintroduced, but also to simplify development and evolution of the code, which you cannot easily do if `-O2` is used for them. Unless a better solution is provided for such a use case, this will stay as is.